setup/install-openroad.sh

```diff
@@ -1,36 +1,19 @@
 #!/bin/sh
 src_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
 
```

Comment:
So all the stuff in our install file is no longer needed?

Reply:
OpenROAD has their own dependency installer and builder, which supports several OSs so it's easier to simply lean on theirs instead